logdna-agent-ds.yaml

```diff
@@ -8,6 +8,10 @@ spec:
       labels:
         app: logdna-agent
     spec:
+      tolerations:
```

Comment:
I would like to propose making this even simpler.

Ideally, we should probably default to sendings logs from ALL boxes in our kubernetes cluster.

To do that, this simply needs to read:

```      
tolerations:
      - operator: Exists
```

Per the k8s docs: An empty key with operator Exists matches all keys, values and effects which means this will tolerate everything.